lib/cdo/workshop_constants.rb

```diff
@@ -0,0 +1,35 @@
+module WorkshopConstants
```

Comment:
Do you know why WorkshopConstants is even in shared? (it looks like you have not made any changes outside of dashboard?)  Are there changes you need to make in pegasus or would it be cleaner just to have this in dashboard (eg. in the Workshop model itself?)